### PR TITLE
Enable FTP/SFTP storage

### PR DESCRIPTION
**Pull request by madwort**

### CODE REVIEW

**Comment by chr4** [comment]
This looks good to me. Would you mind adding a section to the README, explaining howto use the new attribute? (Similar to e.g. `swift_password`).

**Comment by madwort** [comment]
I added https://github.com/chr4-cookbooks/duplicity_ng/pull/42/files#diff-04c6e90faac2675aa89e2176d2eec7d8R139 - is there somewhere else to add it as well? Also noticed that I missed the `unset ...` section.

**Comment by chr4** [comment]
No that's fine. Sorry must have missed the line.

File: README.md [hunk 1]
```diff
@@ -134,6 +134,10 @@ duplicity_ng_cronjob 'myduplicity' do
   azure_account_name 'MY_ACCOUNT_NAME'
   azure_account_key  'MY_ACCOUNT_KEY'
 
+  # In case you use FTP/SFTP as your backend, your credentials go here
+  # Nb. username goes in the url like sftp://username@sftp.mycompany.com/
```
Comment (chr4) [inline README.md]: Would you mind changing `Nb.` to `Note:`? As a non-native, I had to look up what it meant.

Comment (madwort) [inline README.md]: sure!

**Comment by chr4** [comment]
Looks good to me. Thanks for the contribution!

**Comment by madwort** [comment]
👍 Could you do a version bump too while you are online? Cheers!

**Comment by chr4** [comment]
I'm on it, let me fix travis first. Will release new version afterwards.

**Comment by madwort** [comment]
awesome, thanks!

**Comment by chr4** [comment]
`v1.3.0` released. Thanks again!

**Pull request merged**